pkgs/development/python-modules/cirq-rigetti/default.nix

```diff
@@ -1,46 +1,41 @@
 {
+  python,
+  fetchPypi,
   buildPythonPackage,
-  cirq-core,
   setuptools,
-  pyquil,
-  qcs-sdk-python,
-  pytestCheckHook,
+  numpy_1,
+  cirq-core,
 }:
 
+# Cirq-Rigetti is deprecated and limited to python 3.12 and Numpy 1
+# per https://github.com/quantumlib/Cirq/releases/tag/v1.5.0
+# For more details or to provide feedback see https://github.com/quantumlib/Cirq/issues/7058
+
 buildPythonPackage rec {
   pname = "cirq-rigetti";
-  pyproject = true;
-  inherit (cirq-core) version src;
-
-  sourceRoot = "${src.name}/${pname}";
+  version = "1.5.0";
+  format = "wheel";
 
-  pythonRelaxDeps = [ "pyquil" ];
+  disabled = (python.pythonVersion != "3.12");
 
-  postPatch = ''
-    # Remove outdated test
-    rm cirq_rigetti/service_test.py
-  '';
+  src = fetchPypi {
```

Comment:
What's wrong with using https://github.com/quantumlib/Cirq/tree/v1.5.0/cirq-rigetti?

Reply:
Let me see if I can make that work.

Reply:
I'm reverting that change in a moment.

Reply:
Done.